src/components/Element/ArticleElement/ArticleContext.jsx

```diff
@@ -6,10 +6,16 @@ export const useArticleContext = () => useContext(ArticleContext);
 export function ArticleProvider({ children }) {
   const [message, setMessage] = useState('');
   const [sentence, setSentence] = useState({});
+  const [wordList, setWordList] = useState([]);
 
-  return (
-    <ArticleContext.Provider value={{ message, setMessage, sentence, setSentence }}>
-      {children}
-    </ArticleContext.Provider>
-  );
+  function addWord() {
+    const { word } = sentence;
+    const item = { title: word };
+
+    setWordList((list) => list.concat(item));
+  }
+
+  const value = { message, setMessage, sentence, setSentence, wordList, setWordList, addWord };
```

Comment:
if we are adding words through addWord(), do we need to export setWordList?

Reply:
so just export addWord and remove setWordList.

Reply:
i change addWord position, keep only useState in this provider